packages/editor/src/components/provider/index.js

```diff
@@ -102,6 +109,7 @@ class EditorProvider extends Component {
 			__experimentalReusableBlocks: reusableBlocks,
 			__experimentalMediaUpload: hasUploadPermissions ? mediaUpload : undefined,
 			__experimentalFetchLinkSuggestions: fetchLinkSuggestions,
+			__experimentalCanUserUseUnfilteredHTML: canUserUseUnfilteredHTML,
```

Comment:
Any reason you think this shhould be experimental? 
Also these settings are documented in the `defaults.js` file of the block-editor package in addition to providing the default value.

Reply:
Hi @youknowriad the default.js file was updated, nice catch.
The setting was only marked as experimental because lately all new additions were kept experimental for one version so I followed the same logic, but I can change it.